sklearn/decomposition/tests/test_nmf.py

```diff
@@ -144,8 +181,61 @@ def test_nmf_fit_close(solver):
     assert pnmf.fit(X).reconstruction_err_ < 0.1
 
 
-@pytest.mark.parametrize("solver", ("cd", "mu"))
+def test_nmf_true_reconstruction():
+    # Test that the fit is not too far away from an exact solution
+    # (by construction)
+    n_samples = 15
+    n_features = 10
+    n_components = 5
+    beta_loss = 1
+    init = "nndsvda"  # FIXME : should be removed in 1.1
```

Comment:
Similarly, should this number be updated?
```suggestion
    init = "nndsvda"  # FIXME : should be removed in 1.3
```

Reply:
same